package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "resturant",
+  "version": "1.0.0",
+  "description": "A resturant project",
+  "main": "index.js",
+  "scripts": {
+    "dev": "webpack --mode development"
+  },
```

Comment:
Kindly improve your configuration for your application: ⬇️ 
`"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
`
You need to add all those fields. 
![scripts_requirement (1)](https://user-images.githubusercontent.com/32361963/107015318-a9296e80-67ad-11eb-8512-d5e78416e69d.png)